### ISSUE TITLE
Investigate issues with adding users from Active Directory/LDAP

### ISSUE THREAD

**Issue by nwmac** [rancher/dashboard#6290]
Some reports that the user information shown is not correct and it is not easy to search for users, unless the exact text for the username is used.

**Comment by jbilliau-rcd** [comment]
To put some detail behind this, we currently have users duplicated in Rancher quite often. Now sure if it's Rancher doing it, or users doing it, but here is an example:

![image](https://user-images.githubusercontent.com/69006307/178552348-62057fc2-2e7b-4400-b6ce-d8b6145fe0ef.png)

Note this person exists 4 times. Now, if I click the 3-dot menu and click "view in api", i can see the Ping User they are associated with and by finding the one with $firstinitialLastName, we know who the "real" user is.

```
"principalIds": [ 2 items
"ping_user://SAgrawal",
"local://u-luipkckxfi"
],
```

Here is a duplicate:

```
"principalIds": [ 2 items
"ping_user://Agrawal, Suruchi",
"local://u-cifqwmzvqd"
],
```

What happens often, and might be the cause, is I can add users via the dropdown like this:

![image](https://user-images.githubusercontent.com/69006307/178552931-7bd2a6a4-4440-4ea5-b817-86e09cf4fe6b.png)

I hit enter, and it creates that user! But in my opinion, it shouldn't....do a lookup for user based off my text search and if it isn't found, don't give me any results! Instead, users click the name thinking it's correct and in reality, they are exacerbating a duplicate user issue and now, when they go to add that user to a project, it shows up 2, 3, 4 times or more, depending how many "attempt" were made at creating it.

TL;DR - When adding users to clusters/project, do a search based off the text entered into the box and ONLY return results from actual users in the system. If using SAML federated users (which we are, via Ping), I don't want the ability for users to ad-hoc create users on the fly. If it's not connected to Ping, I don't want it.

**Comment by kwwii** [comment]
Can someone please provide details on how to recreate this bug?

**Comment by kwwii** [comment]
This sounds like a bug in their setup. As I cannot reproduce this issue I will close this. Feel free to reopen with any new relevant information.
